lib/ROCrates.Net/Data/update.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Update version number from https://schema.org/docs/releases.html
+curl -L -f -o schema.jsonld https://schema.org/version/10.0/schemaorg-all-http.jsonld
+# Apache License 2.0 https://github.com/schemaorg/schemaorg/blob/V10.0-release/LICENSE
+# no NOTICE - so just keeping file as-is is sufficient (we are also Apache-2.0)
```

Comment:
do we use this shell script? If so, what for? Can't we do this natively with httpclient?

Reply:
This bit, along with the 2 schema files in Data, I nicked from the [python library](https://github.com/ResearchObject/ro-crate-py/tree/master/rocrate/data) just in case we needed them. I don't really know yet whether we do or don't need it